repository-data/webfiles/src/main/resources/site/src/scss/rebrand/components/_api-catalogue.scss

```diff
@@ -50,11 +50,13 @@
     }
   }
 
-  input.section-folder:not(:checked) {
-
-    ~ label .icon svg {
-      rotate: -90deg;
+  input.section-folder:checked {
+    ~ .nhsd-m-filter-menu-section__menu-button label .icon svg {
+      rotate: 180deg;
     }
+  }
+
+  input.section-folder:not(:checked) {
```

Comment:
these can be written in a single `input.section-folder {...}` block by using `&:checked`, `&:not(:checked)` as your nested selectors

Reply:
Another good spot! Fixing now.